### DISCUSSION TITLE
Tag episodes when they are the season finale

### DISCUSSION THREAD

**Discussion by Matth7878** [trakt/showly#603] [Ideas / Feature Requests]
I don't know if it's possible but it would be nice in progress view to tag episodes when they are the season finale.
When looking at next episode to watch or marking one as watched you would know there won't be another one till next season. Currently you can't tell without going to show details.

**Comment by michaldrabik** [comment]
This is a little bit tricky as Trakt does not mark episodes as finale explicitly. This means that when I get the list of episodes for the newest season there is no way of telling if the last one on that list is really the last one of the season or the rest of them have not been added into database/api yet.

**Comment by nafarinha** [comment]
Hi Mike!

Any updates on this? Has Trakt.tv provided any feature that could help implement a season finale tag? The calendar tab has a very useful "season premiere" tag. A "season finale" (or even a "series finale") tag would be very handy.

**Comment by ffminus** [comment]
We wait for whole seasons to be out so we can watch them in one go. Even a best-effort "season finale as of now 🤷‍♂️ " filter would be a nice addition.

Any idea how prevalent the "information trickle" form of publication is vs "whole season dumps"?
